### ISSUE TITLE
SchemaTest failed

### ISSUE THREAD

**Issue by RingerJK** [mapbox/mapbox-navigation-android#2923]
`SchemaTest`, `fun checkNavigationFeedbackEventSize` failed. 
Output: 
```

java.lang.AssertionError: 
Expected :66
Actual   :65
<Click to see difference>


	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at com.mapbox.navigation.core.telemetry.events.SchemaTest.checkNavigationFeedbackEventSize(SchemaTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:68)
	at com.intellij.rt.execution.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:47)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:242)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:70)
```

CI [build](https://app.circleci.com/pipelines/github/mapbox/mapbox-navigation-android/6372/workflows/7da8c8a6-60fa-475c-9e22-fbd84f71a4aa/jobs/24766/steps)

**Comment by RingerJK** [comment]
I've got the same failing on `master` brunch

**Comment by RingerJK** [comment]
Schema has been changed see `event-schema` repo, pr `#112`

**Comment by RingerJK** [comment]
cc @abhishek1508 @Guardiola31337

**Comment by abhishek1508** [comment]
Will look at it cc @RingerJK

**Comment by kmadsen** [comment]
Unblocking our PRs here github.com/mapbox/mapbox-navigation-android/pull/2925

@abhishek1508 will you remove the todos and ignore when this is fixed?

**Comment by abhishek1508** [comment]
Thanks @kmadsen for taking care of this. Yes we will add proper fix for this test once feedback changes are in effect.

**Comment by abhishek1508** [comment]
Closing the issue here since it has been temporarily fixed. It will be fixed for good once [#295](https://github.com/mapbox/navigation-sdks/issues/295) is complete.
